docs/source/install/Developers.rst

```diff
@@ -0,0 +1,144 @@
+Some operating systems and developer environments explicitly require a configuration to be set, which is why the :code:`--config` flag is also passed to the build command. The flag :code:`--parallel <NUMBER_OF_THREADS>` may be added to trigger a parallel build.
+
+Building the project this way generates
+
+- the heuristic library :code:`libqmap_heuristic_lib.a` (Unix) / :code:`qmap_heuristic_lib.lib` (Windows) in the :code:`build/src` directory
+- the heuristic mapper commandline executable :code:`qmap_heuristic` in the :code:`build/apps` directory (only available if Boost is found)
+- a test executable :code:`qmap_heuristic_test` containing a small set of unit tests for the heuristic mapper in the :code:`build/test` directory
+- the exact library :code:`libqmap_exact_lib.a` (Unix) / :code:`qmap_exact_lib.lib` (Windows) in the :code:`build/src` directory (only available if Z3 is found)
+- the exact mapper commandline executable :code:`qmap_exact` in the :code:`build/apps` directory (only available if Boost and Z3 is found)
+- a test executable :code:`qmap_exact_test` containing a small set of unit tests for the exact mapper in the :code:`build/test` directory (only available if Z3 is found)
+
+Working on the Python module
+############################
+
+The :code:`mqt.qmap` Python module can be conveniently built locally by calling
+
+    .. code-block:: console
+
+        (venv) $ pip install --editable .[dev]
+
+The :code:`--editable` flag ensures that changes in the Python code are instantly available without re-running the command. The :code:`[dev]` extra makes sure that all dependencies for running the Python tests and building the documentation are available.
+
+.. note::
+    When using the :code:`zsh` shell it might be necessary to add double quotes around the :code:`.[dev]` part of the command.
+
+`Pybind11 <https://pybind11.readthedocs.io/>`_ is used for providing bindings of the C++ core library to Python (see `bindings.cpp <https://github.com/cda-tum/qmap/tree/main/mqt/qmap/bindings.cpp>`_).
+If parts of the C++ code have been changed, the above command has to be run again to make the changes visible in Python.
+
+Running tests
+#############
+
+C++ core library
+----------------
+
+The C++ part of the code base is tested by unit tests using the `googletest <https://google.github.io/googletest/primer.html>`_ framework.
+The corresponding test files can be found in the :code:`test` directory. In order to build the tests, CMake first has to be configured with the :code:`-DBUILD_QMAP_TESTS=ON` flag, i.e.,
+
+    .. code-block:: console
+
+        $ cmake -S . -B build -DCMAKE_BUILD_TYPE=Release -DBUILD_QMAP_TESTS=ON
+
+Then, the test executables :code:`qmap_heuristic_test` and :code:`qmap_exact_test` is built in the :code:`build/test` directory by calling
+
+    .. code-block:: console
+
+        $ cmake --build build --config Release --target qmap_test
```

Comment:
```suggestion
        $ cmake --build build --config Release --target qmap_heuristic_test
        $ cmake --build build --config Release --target qmap_exact_test
```

Reply:
A misunderstanding on how to build the tests on my part. Updated accordingly.